kernel/src/hil/radio.rs

```diff
@@ -118,3 +118,73 @@ pub trait RadioData<'a> {
         frame_len: usize,
     ) -> Result<(), (ErrorCode, &'static mut [u8])>;
 }
+
+#[derive(PartialEq, Debug, Copy, Clone)]
+pub enum RadioChannel {
+    DataChannel11 = 5,
```

Comment:
I know you just copied this, but since we're adding it to the HIL, why not just `Channel11`?